src/dispatch/project/models.py

```diff
@@ -22,7 +22,7 @@ class Project(Base):
     color = Column(String)
 
     annual_employee_cost = Column(Integer, default=650000)
-    business_year_hours = Column(Integer, default=2080)
+    business_year_hours = Column(Integer, default=1640)
```

Comment:
I'd suggest we leave the default to 2080, as this is the standard, and set 1640 to all our projects' settings in the Web UI. Dispatch gets these values from the project (see [here](https://github.com/Netflix/dispatch/blob/master/src/dispatch/incident_cost/service.py#L187))

Reply:
Permissions? Only organization owners and managers can create projects, and organization owners and managers and project admins can update projects

Reply:
Ah, by this, I just meant that we should have some way to retain the "wisdom" around `1640` versus `2080`. Otherwise it's just in our heads. So in the future, if we're no longer here for some reason, and if it's not enforced by the software. we should have documentation saying "when you create a new project, you shouldn't use the default of 2080 hours per year. We've decided here that 2080 hours is incorrect for us; you should use 1640." That seems like internal Netflix documentation, so I'll add it.